src/main/java/org/spdx/storage/listedlicense/LicenseJsonTOC.java

```diff
@@ -238,4 +239,9 @@ public void setLicenseListVersion(String licenseListVersion) {
 	public void setReleaseDate(String releaseDate) {
 		this.releaseDate = releaseDate;
 	}
+
+	private static String toAbsoluteURL(String relURL) {
+		String retval = relURL.startsWith("./") ? relURL.substring(2) : relURL;
+		return SpdxConstants.LISTED_LICENSE_NAMESPACE_PREFIX + retval;
+	}
```

Comment:
I'm thinking we could make this `https://` rather than `http` since it is referring to a URL and not an RDF property or term.  You can just add a constant string to this file.

BTW - the protocol changed from http to https some years ago.  Since RDF depends on the exact string, we were not able to update the namespace variable to match the URL's.

Reply:
I agree that https should be encouraged for security reasons. I used `SpdxConstants.LISTED_LICENSE_URL` as a new prefix, tell me if there is a problem😃